### DISCUSSION TITLE
Trying to figure out how to `scan` from `RedisPool`

### DISCUSSION THREAD

**Discussion by abcd-ca** [aembke/fred.rs#183] [Q&A] [answered]
Hi, I have a `client` variable which is of type `RedisPool`. I'm trying to call `scan` so that I can delete all keys that start with `foo` (`"foo*"`). It looks like I could call `scan` if my `client` was `RedisClient` but not if it's `RedisPool` but I'm not sure. I'd appreciate someone pointing me in the right direction. I'm expecting a lot of concurrent requests so that's why I'm going for a `RedisPool` over `RedisClient`.

**Comment by aembke** [comment, marked as answer]
@abcd-ca I added a more comprehensive scanning example here: https://github.com/aembke/fred.rs/blob/31f8f56dbb74cb8f750d361529383036e6f66083/examples/scan.rs#L50-L92 

Hopefully that helps, but if you have any more questions feel free to reach out.

**Reply by abcd-ca** [reply]
Thank you for the follow-up :)
